data-raw/paper/paper.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Stacked Ensemble Modeling with Tidy Data Principles'
+tags:
+  - data science
+  - tidyverse
+  - model stacking
+  - ensembling
+  - R
+authors:
+- name: Simon P. Couch
+  orcid: 0000-0001-5676-5107
+  affiliation: "1, 2"
+- name: Max Kuhn
+  affiliation: 2
+affiliations:
+ - name: Johns Hopkins, Department of Biostatistics
+   index: 1
+ - name: RStudio PBC
+   index: 2
+citation_author: Couch et. al.
+date: 10 September 2021
+year: 2021
+bibliography: paper.bib
+output: 
+  rticles::joss_article:
+    keep_tex: true
+    includes:
+      in_header: columns.tex
+csl: apa.csl
+journal: JOSS
+---
+
+# Summary
+
+Model stacking is an ensemble modeling technique that involves training a model to combine the outputs of many constituent statistical models. {stacks} is a free and open-source R software package for stacked ensemble modeling that aligns with tidy data principles. The package’s functionality is closely aligned with the {tidymodels}, a collection of packages providing a unified interface to diverse statistical modeling techniques. Beyond simply providing a mathematically robust interface to build stacked ensemble models, {stacks} adheres to a consistent grammar to interface with two object classes that promote an intuitive understanding of the underlying implementation.
+
+# Statement of Need
+
+*Ensemble learning* (otherwise referred to as *model stacking*) is an increasingly popular approach in statistical and machine learning that involves training a models to generate predictions informed by many constituent models ("members"). Model stacking has been shown to increase predictive performance in a variety of settings and has thus greatly risen in popularity in recent years. In a spring 2020 community survey, the advanced statistical modeling technique was ranked as the first priority for future development among users of the {tidymodels}, a burgeoning software ecosystem for tidyverse-aligned predictive and inferential modeling used across many modern research and industrial applications [@kuhn2022tidymodels]. {stacks} introduces model stacking to the {tidymodels}.
```

Comment:
> *Ensemble learning* (otherwise referred to as *model stacking*

I would repharse this a bit. It makes it sound like there are no other ways to make ensembles. Maybe "one method to create an ensemble model" or similar.

Reply:
You got it👍 [ab2b729](https://github.com/tidymodels/stacks/pull/98/commits/ab2b729086fa36c0ed1667aff1147c08e6c7be8c)